src/main/java/com/zendesk/maxwell/replication/BinlogConnectorReplicator.java

```diff
@@ -186,6 +186,9 @@ private RowMapBuffer getTransactionRows() throws Exception {
 				case XID:
 					buffer.setXid(event.xidData().getXid());
 
+					// feed metric gauge.
+					replicationLag = event.getEvent().getHeader().getTimestamp() - (System.currentTimeMillis() / 1000);
```

Comment:
I tried this and it's milliseconds for me. Can you double check on this? 

![screen shot 2017-04-20 at 9 19 36 am](https://cloud.githubusercontent.com/assets/489032/25206362/e044be90-25aa-11e7-8d84-812bb31419b4.png)